src/tile.js

```diff
@@ -80,12 +80,17 @@ export default class Tile {
     }
 
     freeResources() {
+
         if (this != null && this.meshes != null) {
             for (var p in this.meshes) {
                 this.meshes[p].destroy();
             }
         }
         this.meshes = {};
+
+        if (this.request != null) {
+            this.request.abort();
```

Comment:
If `freeResources` is only ever called on the main thread, then I'm pretty sure we can leave this out. This is one of those tensions around the main thread/worker thread relationship and how the object state is "split" between the two, so this is just another one of those places where it's a little weird (and could possibly be replaced with a better global pattern).